src/server/server.js

```diff
@@ -0,0 +1,64 @@
+
+/**
+ * Dependencies.
+ */
+
+var express = require('express');
+var React = require('react');
+var serializeJavascript = require('serialize-javascript');
+
+var fluxApp = require('../app/app');
+var basePage = React.createFactory(require('../app/base'));
+var navigateAction = require('../app/nav/navigate-action');
+
+/**
+ * Create an express application.
+ */
+
+var expressApp = express();
+
+/**
+ * Serve static assets from the generated 'public' folder.
+ */
+
+expressApp.use(express.static('dist/public'));
+
+/**
+ * Serve application page requests.
+ */
+
+expressApp.use(function (req, res, next) {
+  var context = fluxApp.createContext();
+  context.executeAction(navigateAction, { url: req.url }, function (result) {
+    if (result && result.failed) {
+      next();
```

Comment:
When can an action fail? Looks sketchy -- should you at least log something if it fails?